[NEW] Replace message cog for vertical menu

karlprieb opened the pull request:
<!-- INSTRUCTION: Your Pull Request name should start with one of the following tags -->
<!-- [NEW] For new features -->
<!-- [FIX] For bug fixes -->
<!-- [BREAK] For pull requests including breaking changes -->

<!-- INSTRUCTION: Keep the line below to notify all core developers about this new PR -->
@RocketChat/core 

![image](https://user-images.githubusercontent.com/9200155/29689507-d3570942-88f9-11e7-947c-9587d7ba9d8f.png)

![image](https://user-images.githubusercontent.com/9200155/29689520-de78621c-88f9-11e7-8798-89b3f4a6a702.png)

![image](https://user-images.githubusercontent.com/9200155/29689529-e49495b2-88f9-11e7-9e08-5a2afc1c7e95.png)

![image](https://user-images.githubusercontent.com/9200155/29689535-eb17e65a-88f9-11e7-8a1a-a691791fdb07.png)


<!-- INSTRUCTION: Inform the issue number that this PR closes, or remove the line below -->


<!-- INSTRUCTION: Tell us more about your PR with screen shots if you can -->

graywolf336 commented:
@karlprieb looking good! I see a few things changed with how message actions are defined. Can you give an overview or description? As I see `group` but I am left with a few questions. What are valid values for it? What's it for?

I ask because these changes directly affect Rocketlets and we was just getting started on the message action buttons.

karlprieb commented:
@graywolf336 Thanks! The changes on `messageActions` are simple, I changed `i18nLabel` to `label` and `validation` to `condition` to maintain the standard of nomenclature used on other packages and added `group`.
As you can see on the comment `group` can be `menu` or `message`.

If group is set to `menu` it will be added to this vertical menu, if group is `message` it will appear on the message (next to menu icon that opens the vertical menu).

graywolf336 commented:
@karlprieb Perfect, thanks for the answers! :+1:
